### DISCUSSION TITLE
Yourls Failing with Too Many Redirects

### DISCUSSION THREAD

**Discussion by EchoesFromBeyond** [YOURLS/YOURLS#3482] [Q&A]
Hi there. I have attempted a lot to fix this issue. I am using OVH, I have a subdomain that i'm hosting my YOURLS install at. I believe my OVH is using Apache + NGINX and I have attempted both changing the HTTP and HTTPS Apache Directives to include `ErrorDocument 403 /yourls-loader.php` as well as putting this in the NGINX Directives and neither of these worked:

`if (!-e $request_filename) {
    set $test P;
}
if ($uri !~ ^/(plesk-stat|plesk-git|webstat|webstat-ssl|ftpstat|anon_ftpstat|awstats-icon|internal-nginx-static-location)) {
    set $test "${test}C";
}
if ($test = PC) {
    rewrite ^/(.*)$ /yourls-loader.php?$1;
}`.

When I do not use the NGINX directives and just the Apache ones I get:

`This page isn’t working my.domain redirected you too many times.
[Try clearing your cookies](https://support.google.com/chrome?p=rl_error&hl=en-US).
ERR_TOO_MANY_REDIRECTS`

I have also followed this guide `https://www.danhendricks.com/2018/10/installing-yourls-on-plesk-shared-hosting-domain/`

I also tried the 3 other posts on these 'forums' regarding this topic. Sadly no bueno so far.

Any guidance for other things I could try. I can say that /admin works fine and URL shortening works fine as well from the /admin/ interface.

**Comment by dgw** [comment]
Using Apache is pretty much mutually exclusive from using nginx. Your site cannot be served by both, because only one or the other can listen on the required ports.

You need to determine which HTTP server you are using, and apply the relevant configuration for it.
